api/scpca_portal/models/job.py

```diff
@@ -158,21 +158,20 @@ def get_sample_job(
         )
 
     @classmethod
-    def create_terminated_retry_jobs(cls) -> List[Self]:
+    def create_terminated_retry_jobs(cls, terminated_jobs: List[Self]) -> List[Self]:
```

Comment:
Maybe this can become `create_retry_jobs` since we don't care about if it was terminated or not.